Control coverage/Feature/VirtualMachineScaleSet.md

```diff
@@ -739,4 +740,77 @@ Using this feature ensures that sensitive data is stored encrypted at rest. This
   <br />
  <br />
 
+___
+
+## Azure_VirtualMachineScaleSet_AuthN_Enable_AAD_Auth_Linux_Trial
+
+### Display Name
+[Trial] AAD extension must be deployed to the Linux VMSS
+
+### Rationale
+Installing AAD extension on VMSS allows you to login into VMSS instances using Azure AD, making it possible to login user without password and improves authentication security.
+
+### Control Settings 
+```json 
+{
+    "Linux": {
+          "ExtensionType": "AADSSHLoginForLinux",
+          "ExtensionPublisher": "Microsoft.Azure.ActiveDirectory",
+          "ProvisioningState": "Succeeded"
+        }
+}
+ ``` 
+### Note:
+This control only covers Virtual Machine Scale Sets with 'Uniform' Orchestration mode and following Linux distributions are currently supported for deployments in a supported region.
+
+> 1. Common Base Linux Mariner (CBL-Mariner) - CBL-Mariner 1, CBL-Mariner 2
+> 2. CentOS - CentOS 7, CentOS 8
+>3. Debian - Debian 9, Debian 10, Debian 11
+>4. openSUSE - openSUSE Leap 42.3, openSUSE Leap 15.1+
+>5. RedHat Enterprise Linux (RHEL) - RHEL 7.4 to RHEL 7.10, RHEL 8.3+
+>6. SUSE Linux Enterprise Server (SLES) - SLES 12, SLES 15.1+
+>7. Ubuntu Server - Ubuntu Server 16.04 to Ubuntu Server 22.04
+
+
+### Control Spec
+
+> **Passed:**
+> AAD Extension is present for Linux Virtual Machine Scale Set with provisioning state as succeeded.
+>
+> **Failed:**
+> AAD Extension is missing or provisioning state is not succeeded.
+>
+> **NotScanned:**
```

Comment:
I think we discussed on this, this should be error scenario.

Reply:
Yes I have updated this part.